src/intervals/functions.jl

```diff
@@ -18,68 +18,71 @@ function ^(a::Interval{BigFloat}, n::Integer)
     isempty(a) && return a
     n == 0 && return one(a)
     n == 1 && return a
-    n == 2 && return sqr(a)
+    # n == 2 && return sqr(a)
     n < 0 && a == zero(a) && return emptyinterval(a)
 
     T = BigFloat
 
     if isodd(n) # odd power
         isentire(a) && return a
         if n > 0
-            a.lo == zero(T) && return @round(T, zero(T), a.hi^n)
-            a.hi == zero(T) && return @round(T, a.lo^n, zero(T))
-            return @round(T, a.lo^n, a.hi^n)
+            a.lo == 0 && return @round(0, a.hi^n)
+            a.hi == 0 && return @round(a.lo^n, 0)
+            return @round(a.lo^n, a.hi^n)
         else
-            if a.lo ≥ zero(T)
-                a.lo == zero(T) && return @round(T, a.hi^n, convert(T, Inf))
-                return @round(T, a.hi^n, a.lo^n)
+            if a.lo ≥ 0
+                a.lo == 0 && return @round(a.hi^n, Inf)
+                return @round(a.hi^n, a.lo^n)
 
-            elseif a.hi ≤ zero(T)
-                a.hi == zero(T) && return @round(T, convert(T, -Inf), a.lo^n)
-                return @round(T, a.hi^n, a.lo^n)
+            elseif a.hi ≤ 0
+                a.hi == 0 && return @round(-Inf, a.lo^n)
+                return @round(a.hi^n, a.lo^n)
             else
                 return entireinterval(a)
             end
         end
 
     else # even power
         if n > 0
-            if a.lo ≥ zero(T)
-                return @round(T, a.lo^n, a.hi^n)
-            elseif a.hi ≤ zero(T)
-                return @round(T, a.hi^n, a.lo^n)
+            if a.lo ≥ 0
+                return @round(a.lo^n, a.hi^n)
+            elseif a.hi ≤ 0
+                return @round(a.hi^n, a.lo^n)
             else
-                return @round(T, mig(a)^n, mag(a)^n)
+                return @round(mig(a)^n, mag(a)^n)
             end
 
         else
-            if a.lo ≥ zero(T)
-                return @round(T, a.hi^n, a.lo^n)
-            elseif a.hi ≤ zero(T)
-                return @round(T, a.lo^n, a.hi^n)
+            if a.lo ≥ 0
+                return @round(a.hi^n, a.lo^n)
+            elseif a.hi ≤ 0
+                return @round(a.lo^n, a.hi^n)
             else
-                return @round(T, mag(a)^n, mig(a)^n)
+                return @round(mag(a)^n, mig(a)^n)
             end
         end
     end
 end
 
 function sqr{T<:Real}(a::Interval{T})
-    isempty(a) && return a
-    if a.lo ≥ zero(T)
-        return @round(T, a.lo^2, a.hi^2)
-
-    elseif a.hi ≤ zero(T)
-        return @round(T, a.hi^2, a.lo^2)
-    end
-
-    return @round(T, mig(a)^2, mag(a)^2)
+    return a^2
+    # isempty(a) && return a
+    # if a.lo ≥ zero(T)
+    #     return @round(a.lo^2, a.hi^2)
+    #
+    # elseif a.hi ≤ zero(T)
+    #     return @round(a.hi^2, a.lo^2)
+    # end
+    #
+    # return @round(mig(a)^2, mag(a)^2)
 end
 
+^(a::Interval{BigFloat}, x::AbstractFloat) = a^big(x)
```

Comment:
Yet, our tests are failing right here: the test checks that `x^(1//3) --> x^(1/3)` holds, for `x = biginterval(27)`; the correct answer *must* contain 3. So this is the method that is being called for the RHS. 

Checking this test, `big(1//3)` behaves correctly (renders `BigInt(1)//BigInt(3)`), and the answer contains `3`. However, `big(1/3)` obviously behaves incorrectly (converts the `Float64` representation of `1/3` to `BigFloat`, which is far away from 1/3!).  Incidentally, my proposal is not working either for this.